pyannote/metrics/base.py

```diff
@@ -247,6 +247,21 @@ def __iter__(self):
         for uri, component in self.results_:
             yield uri, component
 
+    def __add__(self, other):
+        cls = self.__class__
+        result = cls()
+        result.results_ = self.results_ + other.results_
+        for cname in self.components_:
+            result.accumulated_[cname] += self.accumulated_[cname]
+            result.accumulated_[cname] += other.accumulated_[cname]
+        return result
+
+    def __radd__(self, other):
+        if other == 0:
```

Comment:
Is this how the built-in `sum` function initializes its own accumulator? 
Would be nice to add a quick comment mentionning this...

Reply:
The built-in `sum` actually has a `start` parameter that controls the initial value of the summation. As you might gather, the default value is for `start` is 0, so I just hard coded that value as an additive identity for metrics. Probably would be good to add a one or two line comment to this effect to save someone having to read up on `__radd__` and `sum`.